src/lib/filter-versions.ts

```diff
@@ -0,0 +1,61 @@
+import semverSatisfies from 'semver/functions/satisfies'
+import semverMajor from 'semver/functions/major'
+import semverMinor from 'semver/functions/minor'
+import semverPatch from 'semver/functions/patch'
+import { ReleasesAPIResponse } from 'lib/fetch-release-data'
+
+/**
+ * Pulled from terraform-website/pages/downloads/index.jsx on 2022-03-09:
+ * https://github.com/hashicorp/terraform-website/blob/master/pages/downloads/index.jsx#L55-L98
+ *
+ * Modified 2022-03-28 to replace semverGte with semverSatisfies,
+ * as the former seemed to return pre-releases, which is not what we want.
+ */
+function filterVersions(
```

Comment:
Moved from this file: https://github.com/hashicorp/dev-portal/pull/2228/files#diff-82193e22f46ec6f8cf4dd14abf1b88cd8d9916bee43269ccb4ab0295839e87bb